lib/hydra/works/models/concerns/generic_work_behavior.rb

```diff
@@ -20,11 +20,48 @@ module GenericWorkBehavior
       type [RDFVocabularies::PCDMTerms.Object,WorksVocabularies::WorksTerms.GenericWork]
       include Hydra::Works::AggregatesGenericFiles
       include Hydra::Works::AggregatesGenericWorks
+      include Hydra::Works::BlockChildObjects
+
+      filters_association :members, as: :child_generic_works, condition: :works_generic_work?
```

Comment:
There is no PCDM::Work.  Is this a typo?

There are lots of use cases for GenericWork as a child of another GenericWork. 

Ex.  https://docs.google.com/document/d/1FxYyhVkjIuvOq1d-MS2dzKTElIfNSk01ytwiAB5DZiw/edit